components/tidb_query/src/util.rs

```diff
@@ -5,6 +5,33 @@ use tipb::ColumnInfo;
 
 use crate::codec::datum::Datum;
 
+const MAX_RAND_VALUE: u32 = 0x3FFFFFFF;
+
+pub struct MySQLRng {
```

Comment:
Prefer to move it to `tidb_query/src/expr_util/rand.rs`. That directory is where other common function implementations placed in.

Reply:
refactored.